---
int64 tot_distancia
int64 evitados_izda
int64 evitados_dcha

